x-pack/test/alerting_api_integration/common/config.ts

```diff
@@ -333,6 +333,7 @@ export function createTestConfig(name: string, options: CreateTestConfigOptions)
               ]
             : []),
           '--notifications.connectors.default.email=notification-email',
+          '--xpack.task_manager.allow_reading_invalid_state=false',
```

Comment:
This is turning it off for all the FT, not just serverless.  Just mentioning as the title seems to indicate it's just for serverless.